site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx

```diff
@@ -283,17 +281,13 @@ export const WorkspaceScheduleForm: FC<WorkspaceScheduleFormProps> = ({
 }
 
 export const ttlShutdownAt = (now: dayjs.Dayjs, workspace: Workspace, tz: string, newTTL: number): string => {
-  const newDeadline = dayjs(workspace.latest_build.updated_at).add(newTTL, "hour")
+  const newDeadline = dayjs(workspace.latest_build.deadline).add(newTTL, "hour")
```

Comment:
@vapurrmaid I'm pretty sure this was a bug

Reply:
Yep, that's a bug